common/src/main/java/com/microsoft/identity/common/internal/authscheme/PopAuthenticationSchemeInternal.java

```diff
@@ -165,16 +152,14 @@ public String getAccessTokenForScheme(@NonNull final String accessToken) throws
         final long ONE_SECOND_MILLIS = 1000L;
         final long timestampMillis = mClockSkewManager.getAdjustedReferenceTime().getTime();
 
-        return Device
-                .getDevicePoPManagerInstance()
-                .mintSignedAccessToken(
-                        getHttpMethod(),
-                        timestampMillis / ONE_SECOND_MILLIS,
-                        getUrl(),
-                        accessToken,
-                        getNonce(),
-                        getClientClaims()
-                );
+        return mPopManager.mintSignedAccessToken(
```

Comment:
I didn't catch this earlier... i think we should try to stick to a limited number of verbs for conveying create.